docs/releases/3.2.txt

```diff
@@ -60,6 +60,12 @@ Minor features
 
 * ...
 
+:mod:`django.contrib.humanize`
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The :tfilter:`intword` and :tfilter:`intcomma` filters do not depend on the
+  :setting:`USE_L10N` setting any longer.
```

Comment:
```
* The :tfilter:`intword` and :tfilter:`intcomma` templates filters no longer depend on the :setting:`USE_L10N` setting.
```
Please move it to the `Backwards incompatible changes in 3.2` -> `Miscellaneous`.